client/assets/components/common/form.scss

```diff
@@ -0,0 +1,42 @@
+.as-form-group {
```

Comment:
What does `as` mean?

Reply:
`andela social`

Reply:
I used it because `.form-group` is a class name common to the 3rd party css we currently have in the project

Reply:
Make sense